rhubarb/src/audio/WaveFileReader.cpp

```diff
@@ -172,6 +176,12 @@ inline AudioClip::value_type readSample(
 				sum += toNormalizedFloat(raw, INT24_MIN, INT24_MAX);
 				break;
 			}
+			case SampleFormat::Int32:
+			{
+				int raw = read<int>(file);
+				sum += toNormalizedFloat(raw, INT_MIN, INT_MAX);
```

Comment:
`INT_MIN` and `INT_MAX` seem to depend on the width of the `int` type, which may be 64 bits on some systems. Could you use 32-bit-specific constants instead?

Reply:
Do you mean INT32_MIN and INT32_MAX?

Reply:
Sounds good.

Reply:
Fixed.  Sorry for taking so long.

Reply:
No problem! In the meantime, I noticed that my CI was broken and I fixed it. Could you rebase your PR onto my master branch? If I made no mistakes, this should trigger proper CI builds on all platforms.